### DISCUSSION TITLE
ECU EKLEME

### DISCUSSION THREAD

**Discussion by Ruygh** [cedricp/ddt4all#1097] [General]
Merhaba Renault TALİANT DDT4ALL  programı ile erişim sağlanamıyor lütfen bu modeli de ekler misiniz?

**Comment by Furtif** [comment]
> Merhaba Renault TALİANT DDT4ALL programı ile erişim sağlanamıyor lütfen bu modeli de ekler misiniz?

Véhicule/code projet
RENAULT TALIANT (LJF)

appears to be a Logan concept 

https://www.tech-accessoires.renault.com/renault-techaxs/media?mediaTitle=title_Fitting%26nbsp%3Binstructions%26nbsp%3Bfor%26nbsp%3Bspoiler%26nbsp%3BLJI%2BLJF-09.04.2021&mediaId=1328101

**Reply by Ruygh** [reply]
Denedim no date yazıyor birde Logan 1 2 3 diye var hepsinde denedim yanlış mı yaptım acaba

**Reply by Furtif** [reply]
I don't know, the only thing I know is that the LJF code is not in databases.

**Reply by Ruygh** [reply]
AYNEN LJF yok XJF var onu denesem olur mu?

**Reply by Furtif** [reply]
![image](https://github.com/cedricp/ddt4all/assets/11375035/53698303-f5eb-4e12-acdb-f0647ad04704)
Maybe a scan all will find the possible ECUs.

**Reply by Ruygh** [reply]
Bende ordan yapıyorum ekran boş gösteriyor siz hangi versiyonu kullanıyorsunuz?

**Reply by Furtif** [reply]
https://github.com/cedricp/ddt4all/releases/tag/v3.0.1

**Reply by Ruygh** [reply]
Ben de bunu indirmişim demek ki data yok

**Reply by Furtif** [reply]
If you have a good database and nothing appears in connected mode, it is possible
Try database:
https://github.com/cedricp/ddt4all/discussions/845#discussioncomment-6144566

**Reply by Ruygh** [reply]
İndirdim program Giles/ddt4all dizi dinle atıyorum fakat attıktan sonra uygulamaya girince 0 date hatası veriyor

**Reply by Furtif** [reply]
Ecu.zip dosyasını açmadan ddt4all'a yerleştirmelisiniz, indirirken zip içinde kalması gerekir, zip'i c:/program files/ddt4all/ dizinine kopyalamanız gerekir veya LinuxOS kullanıyorsanız zip'i kök dizine kopyalamanız gerekir. klonlanmış git'in, Apple MacOsX için aynı.

**Reply by Ruygh** [reply]
Sağolun bu veritabanında da denedim olmadı teşekkürler demek ki data verisinde yok bilgisi, teşekkürler cevap verdiğiniz için, rica etsek eklemezler mi?

**Reply by ramazansancar** [reply]
@Ruygh No Data yazıyorsa ya araç sensöründen veri alınamamış ya da araçta o özellik olmayabilir. Buna ek olarak seçmiş olduğunuz ecu dosyasında sorun olabilir.  Kullanmış olduğunuz bağlantı tipine göre bağlı olduğundan dtt4all sağ alt köşesinde `connected` yazmasından emin olabilirsiniz

**Comment by ysndgn06** [comment]
ecu.zip dosyam yok yardımcı olabilirmisiniz ustam veritabanı dosyasına gerçekten çok ihtiyacımvar

**Reply by mishu28nmv** [reply]
[ECU.ZIP](https://drive.google.com/file/d/1W0zBDvAScA4_p_GNnszUu80sKH5KxxSk/view?usp=drive_link)

**Reply by ramazansancar** [reply]
@mishu28nmv erişim izni açık değil :D

**Reply by 9krs55ykhv-dotcom** [reply]
@Furtif, you should update the link in discussion #845. I had a hard time finding it.

**Reply by Furtif** [reply]
not.

**Reply by Renotech1** [reply]
Still need? Can share link
> @Furtif, you should update the link in discussion #845. I had a hard time finding it.

**Reply by ramazansancar** [reply]
@mishu28nmv Could the file have been accidentally deleted?

"The file is in the owner's trash."
